sklearn/multiclass.py

```diff
@@ -956,10 +956,10 @@ class OutputCodeClassifier(MetaEstimatorMixin, ClassifierMixin, BaseEstimator):
     estimators_ : list of `int(n_classes * code_size)` estimators
         Estimators used for predictions.
 
-    classes_ : numpy array of shape [n_classes]
+    classes_ : array-like of shape (n_classes,)
```

Comment:
this one will be an ndarray

Reply:
My bad. I should have checked again the code, looking for these.